### DISCUSSION TITLE
Making git commits to elements on index.html page

### DISCUSSION THREAD

**Discussion by preston-lai** [LekoArts/gatsby-themes#974] [Questions] [answered]
I'm trying to change an element (the website's title where it says "Cara - Gatsby Starter Portfolio" when opening a new tab) which can be found on public -> index.html  pages, however, when I make an edit and run git status - the change does not show up. I can only see changes committed on the mdx page using the git status command. 

Can anyone help with this?

**Comment by LekoArts** [comment, marked as answer]
Please read https://github.com/LekoArts/gatsby-themes/tree/main/themes/gatsby-theme-cara#shadowing, it explains how to change things in a theme.
